.github/workflows/charm4py.yml

```diff
@@ -17,14 +17,21 @@ jobs:
     - name: build-charm4py
       run: |
         git fetch --unshallow # Need full repo for 'git describe' used by setup.py
-        pip3 install setuptools cython cffi greenlet numpy
+        pip3 install setuptools cython greenlet numpy
         git clone https://github.com/UIUC-PPL/charm charm_src/charm
         export CHARM_EXTRA_BUILD_OPTS="--enable-error-checking"
         export CHARM_BUILD_PROCESSES=2
-        export CHARM4PY_BUILD_CFFI=1
         python3 setup.py build_ext --inplace
+        export PYTHONPATH="$PWD"
     - name: test-charm4py
       run: |
+        git clone https://github.com/slm960323/task-bench.git
```

Comment:
If this repo should be used in CI, it should be under the UIUC-PPL organization or made part of the `charm4py` repo, I think.